tests/test_filter.py

```diff
@@ -80,35 +80,35 @@ def test_append_filtered_policy(self):
     def test_filtered_policy_invalid_filter(self):
         adapter = casbin.persist.adapters.FilteredAdapter(get_examples("rbac_with_domains_policy.csv"))
         e = casbin.Enforcer(get_examples("rbac_with_domains_model.conf"), adapter)
-        filter = ["", "domain1"]
+        test_filter = ["", "domain1"]
 
         with self.assertRaises(RuntimeError):
-            e.load_filtered_policy(filter)
+            e.load_filtered_policy(test_filter)
 
     def test_filtered_policy_empty_filter(self):
         adapter = casbin.persist.adapters.FilteredAdapter(get_examples("rbac_with_domains_policy.csv"))
         e = casbin.Enforcer(get_examples("rbac_with_domains_model.conf"), adapter)
 
         try:
             e.load_filtered_policy(None)
-        except:
+        except Exception:
             raise RuntimeError("unexpected error in LoadFilteredPolicy")
 
         if e.is_filtered():
             raise RuntimeError("adapter did not reset the filtered flag correctly")
 
         try:
             e.save_policy()
-        except:
+        except Exception:
             raise RuntimeError("unexpected error in SavePolicy")
 
     def test_unsupported_filtered_policy(self):
         e = casbin.Enforcer(get_examples("rbac_with_domains_model.conf"), get_examples("rbac_with_domains_policy.csv"))
-        filter = Filter()
-        filter.P = ["", "domain1"]
-        filter.G = ["", "", "domain1"]
+        test_filter = Filter()
```

Comment:
test_filter

Reply:
coz filter is a built-in name in Python, redefining it interferes with Python's syntax system